### PR TITLE
chore: skip cluster tests if redis-server wasn't found

### PR DESCRIPTION
**Pull request by BorysTheDev**

### CODE REVIEW

**Review by chakaz — COMMENTED** [review]
Is there a way to check that this is not running as part of a github runner? Because if it is, we'd need to fix it (i.e. understand why Redis is not found)

**Comment by BorysTheDev** [comment]
I've took this code from another place, but I don't know how to check where script is running. Maybe we can add some parameter.

**Comment by BorysTheDev** [comment]
> Is this of use? https://stackoverflow.com/questions/73973332/check-if-were-in-a-github-action-travis-ci-circle-ci-etc-testing-environme

good catch, I will try it and if it works will fix the previous using of skip tests

**Comment by BorysTheDev** [comment]
@chakaz I've done check that this approach works here https://github.com/dragonflydb/dragonfly/pull/3422

File: tests/dragonfly/cluster_test.py [hunk 1]
```diff
@@ -57,9 +57,14 @@ def redis_cluster(port_picker):
...
-        node.start()
-        time.sleep(1)
+    try:
+        for node in nodes:
+            node.start()
+            time.sleep(1)
+    except FileNotFoundError as e:
+        if os.getenv("GITHUB_ACTIONS") == None:
+            pytest.skip("Redis server not found")
+        raise
```
Comment (chakaz) [inline tests/dragonfly/cluster_test.py:67]: How about creating some `SkipIfNotInGitHub()` or at least `IsRunningInGitHub()`?

**Review by chakaz — APPROVED** [review]

**Pull request merged**